utils/browser/src/lib.rs

```diff
@@ -116,11 +116,6 @@ struct RpcMessage {
 
 /// Create a Client object that connects to a service.
 pub fn start_client(mut service: impl AbstractService) -> Client {
-	// Spawn informant
-	wasm_bindgen_futures::spawn_local(
-		sc_informant::build(&service, sc_informant::OutputFormat::Plain).map(drop)
-	);
-
```

Comment:
Whats the reasoning for removing this? Having an informant on the light client is useful.

Reply:
It's done already in the service builder in the `build()` method

Reply:
(I mean I just moved it in this PR)

Reply:
I just realized that we cannot **not** have an informant anymore... :thinking: is this a problem? cc @bkchr

Reply:
It was before also always enabled by default.